benchmark.go

```diff
@@ -41,6 +42,40 @@ var (
 	errNegativeMaxReqs  = errors.New("max requests cannot be negative")
 )
 
+// Struct to hold values of benchmark parameters
+type BenchmarkParameters struct {
+    CPUs int `json:"CPUs"`
```

Comment:
for naming consistency with Go, can you use `camelCase` for the json tag names? e.g. `cpus`, `maxRequests`, etc.

ref: https://stackoverflow.com/questions/5543490/json-naming-convention